lectures/markov_chains.md

```diff
@@ -980,11 +974,11 @@ for i in range(n_state):
     axes[i].grid()
     axes[i].set_ylim(0.45, 0.55)
     axes[i].axhline(ψ_star[i], linestyle='dashed', lw=2, color = 'black', 
-                    label = fr'$\psi^*(X={i})$')
+                    label = fr'$\psi^*({i})$')
```

Comment:
\psi^*(i)

Reply:
By the way @HumphreyYang , $\psi^*$ is a map on $0, \ldots, n-1$, which is why the change is requested.

Reply:
Many thanks for pointing this out @jstac . I used `X={i}` here mainly to show that this is the probability of $X=1/X=2/X=3$ in the stationary distribution $\psi^*$. Should I instead indicate this using a short title on each graph?

Reply:
Many thanks for pointing this out @jstac. I used `X={i}` here mainly to show that this is the probability of $X=1/X=2/X=3$ in the stationary distribution $\psi^*$. Should I instead indicate this using a short title on each graph?

Reply:
Many thanks for pointing this out @jstac. I used `X={i}` here mainly to show that this is the probability of $X=1/X=2/X=3$ in the stationary distribution $\psi^*$. Should I instead indicate this using a short title on each graph?

Reply:
I understand and I appreciate the effort but it's the wrong notation here, since $\psi^*$ is a map on the integers, not subsets of the sample space.  You would need to use something like `\mathbb P\{X_t = 0\}` and clarify the distribution of $X_t$.

You could add something like that to the text somewhere if you think it helps.

Also, `(i)`, rather than `({i})`.

Reply:
Thank you so much for the meticulous check. I understand your intention to change the symbol. The code

```
label = fr'$\psi^*({i})$')
```

prints out a plot like this: 
![Screenshot 2023-02-08 at 11 45 18](https://user-images.githubusercontent.com/39026988/217399951-0e08b1df-e3f4-434c-ae1b-955221b2a577.png)

which corresponds to the notation $\psi^*(x)$ used in 

![Screenshot 2023-02-08 at 11 44 29](https://user-images.githubusercontent.com/39026988/217400034-8251a17c-5b22-443c-98e5-57608de85077.png)

Reply:
Oh, silly me :-)

Thanks @HumphreyYang for your patience.  You are correct and this was my mistake.  Please ignore.